### PR TITLE
fix on plugin wrapper in order to work with webpack

### PR DESCRIPTION
**Pull request by n1crack**
https://github.com/itsjavi/bootstrap-colorpicker/issues/195

### CODE REVIEW

File: dist/js/bootstrap-colorpicker.js [hunk 1]
```diff
@@ -8,23 +8,21 @@
...
-
-  if (typeof exports === 'object') {
-    module.exports = factory(global.jQuery, global);
-  } else if (typeof define === 'function' && define.amd) {
-    define(['jquery'], function(jq) {
-      return factory(jq, global);
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module unless amdModuleId is set
+    define(["jquery"], function(a0) {
```
Comment (itsjavi) [inline dist/js/bootstrap-colorpicker.js]: what does `a0` mean? can you use a more auto-describing variable name?

File: dist/js/bootstrap-colorpicker.js [hunk 2]
```diff
@@ -8,23 +8,21 @@
...
+      return (factory(a0));
     });
-  } else if (global.jQuery && !global.jQuery.fn.colorpicker) {
-    factory(global.jQuery, global);
+  } else if (typeof exports === 'object') {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory(require("jquery"));
+  } else {
```
Comment (itsjavi) [inline dist/js/bootstrap-colorpicker.js]: why did you delete this part of the condition `else if (!jQuery.fn.colorpicker)` ?
it avoids re-executing the factory if the colorpicker plugin is already loaded.
can you restore this condition?

File: dist/js/bootstrap-colorpicker.js [hunk 3]
```diff
@@ -8,23 +8,21 @@
...
+    factory(jQuery);
   }
-}(function(jQuery, window) {
-  'use strict';
```
Comment (itsjavi) [inline dist/js/bootstrap-colorpicker.js]: you also deleted `use strict` why??

**Comment by n1crack** [comment]
I will update the pr with the needed changes when I get home.

**Comment by n1crack** [comment]
made the changes.

**Comment by itsjavi** [comment]
thank you!
